pyproject.toml

```diff
@@ -100,6 +100,18 @@ help = "Run tests with multiple pythons"
 cmd = "black . --check --diff"
 help = "Check if code style is correct"
 
+[tool.isort]
+py_version = 36
+profile = "black"
+force_single_line = true
```

Comment:
I've personally never seen conflicts on imports especially cause black will put them on their own separate "not-conflict-able" line if it's long enough. I think a potential solution could just be to make all imports end in a trailing comma and let black handle it

Reply:
I think that's unnecessary effort. That said, for the purposes of this PR, I'll remove that config and leave the current status quo so that we can get basic sorting in.